src/helpers/utils.js

```diff
@@ -1,5 +1,8 @@
+/* eslint-disable valid-jsdoc */
```

Comment:
Welldone @tobechukwuobitube , Please remove this comment and obey the `eslint` configuration to maintain consistency in the project

Reply:
Thank you for this feedback.
This could be an oversight from a rebase; I didn't add this file that contained this comment myself. I guess I got it from a rebase.
However, I will look at it and see what can be done about it.